packages/block-editor/src/components/grid-visualizer/grid-item-resizer.js

```diff
@@ -12,14 +13,89 @@ import { getComputedCSS } from './utils';
 
 export function GridItemResizer( { clientId, onChange } ) {
 	const blockElement = useBlockElement( clientId );
+	const rootBlockElement = blockElement?.parentElement;
+	const [ resizeDirection, setResizeDirection ] = useState( null );
+
+	/*
+	 * Resizer dummy is an empty div that exists only so we can
```

Comment:
This doc comment isn't accurate anymore now that we're using `__unstableContentRef`.